publications-2020.bib

```diff
@@ -1,5 +1,16 @@
 % Encoding: US-ASCII
 
+@Article{2020:aggul:defect,
+  author  = { Aggul, Mustafa },
```

Comment:
I'm curious where these extra spaces around the curly brackets come from. Are you using `bibtool 2.68`? (you can check it with `bibtool -V` with a capital `V` - otherwise you run in verbose mode)

Reply:
`doi2bib.py` created these already. `bibtool` just left them in place. (I use bibtool 2.67.)

But can we please not worry about spaces before or after braces? I want this list of entries finished this week, and spaces are really not very high on my list of things I want to worry about :-)